firefox-ios/Client/Frontend/Browser/Toolbars/Redux/AddressBarState.swift

```diff
@@ -13,7 +13,7 @@ struct AddressBarState: StateType, Equatable {
     var browserActions: [ToolbarActionState]
     var borderPosition: AddressToolbarBorderPosition?
     var url: URL?
-    var lockIconImageName: String
+    var lockIconImageName: String?
```

Comment:
Why is the lock icon optional now?

Reply:
As commented below, there is the case when we have to hide it in case reader mode is active.